packages/calypso-e2e/src/lib/pages/my-home-page.ts

```diff
@@ -11,10 +11,29 @@ import { Page } from 'playwright';
 const selectors = {
 	dashboard: '.customer-home__main',
 	statsCard: '.stats',
+	visitSiteButton: 'text=Visit site',
 };
 
+/**
+ * Page representing the WPCOM home dashboard.
+ *
+ * @augments {BaseContainer}
+ */
 export class MyHomePage extends BaseContainer {
 	constructor( page: Page ) {
 		super( page, selectors.dashboard );
 	}
+
+	/**
+	 * Click on the Visit Site button on the home dashboard.
+	 *
+	 * @returns {Promise<void>} No return value.
+	 */
+	async visitSite(): Promise< void > {
+		await this.page.waitForSelector( selectors.visitSiteButton );
+		await Promise.all( [
+			this.page.waitForNavigation(),
+			this.page.click( selectors.visitSiteButton ),
```

Comment:
If `page.click` waits for a selector on its own, do we still need to explicitly wait for it?
```suggestion
		await Promise.all( [
			this.page.waitForNavigation(),
			this.page.click( selectors.visitSiteButton ),
```

Reply:
Similar to the comment left earlier - it may be prudent, but perhaps not strictly necessary.

Reply:
It was not necessary and so I have removed the wait.